source/store.go

```diff
@@ -316,26 +315,24 @@ func NewKubeClient(kubeConfig, kubeMaster string, requestTimeout time.Duration)
 // wrappers) to the client's config at this level. Furthermore, the Istio client
 // constructor does not expose the ability to override the Kubernetes master,
 // so the Master config attribute has no effect.
-func NewIstioClient(kubeConfig string) (*istiocontroller.Client, error) {
-	if kubeConfig == "" {
+func NewIstioClient(kubeconfig string, kubeMaster string) (*istioclient.Clientset, error) {
+	if kubeconfig == "" {
```

Comment:
I would still leave it like it is, IMHO this is more go convenient

Reply:
This has been done. Thanks!